ArcGISRuntimeSDKQt_QMLSamples/Scenes/ChooseCameraController/ChooseCameraController.qml

```diff
@@ -0,0 +1,193 @@
+    property real altitude: 10000
+    property real cameraDist: 5000
+
+    GlobeCameraController {
+        id: globeController
+    }
+
+    Point {
+        id: theCrater
+        x: longitude
+        y: latitude
+        z: altitude
+        spatialReference: SpatialReference { wkid: 4326 }
+    }
+
+    OrbitLocationCameraController {
+        id: orbitLocationCtrlr
+        targetLocation: theCrater
+        cameraDistance: cameraDist
+    }
+
+    OrbitGeoElementCameraController{
+        id: orbitPlaneCtrlr
+        targetGeoElement: thePlane
+        cameraDistance: cameraDist / 2
+    }
+
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+
+        Scene {
+            id: scene
+            BasemapImagery {}
+
+            Surface {
+                ArcGISTiledElevationSource {
+                    url: "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+            }
+        }
+
+        GraphicsOverlay {
+            id: graphicsOverlay
+
+            LayerSceneProperties {
+                surfacePlacement: Enums.SurfacePlacementRelative
+            }
+
+            ModelSceneSymbol {
+                id: mms
+                url: dataPath + "/Bristol/Collada/Bristol.dae"
+                scale: 10.0
+                heading: 0.0
+            }
+
+            Graphic {
+                id: thePlane
+                symbol: mms
+
+                geometry: Point {
+                    x: longitude
+                    y: latitude
+                    z: altitude / 4
+                    spatialReference: sceneView.spatialReference
+                }
+            }
+        }
+
+        Component.onCompleted: {
+            // set viewpoint to the specified camera
+            setViewpointCameraAndWait(camera);
+        }
+    }
+
+    Camera {
+        id: camera
+        heading: 0.0
+        pitch: 0.0
+        roll: 0.0
+
+        Point {
+            x: longitude
+            y: latitude
+            z: altitude
+            spatialReference: SpatialReference { wkid: 4326 }
+        }
+    }
+
+    // Create a rectangle to display the GUI
+    Rectangle {
+        id: layerVisibilityRect
+        anchors {
+            margins: 10
+            left: parent.left
+            top: parent.top
+        }
+        height: 150
+        width: 250
+        color: "transparent"
+
+        Rectangle {
+            anchors.fill: parent
+            width: layerVisibilityRect.width
+            height: layerVisibilityRect.height
+            color: "lightgrey"
+            opacity: .8
+            radius: 5
+            border {
+                color: "#4D4D4D"
+                width: 1
+            }
+
+            Column {
+                anchors {
+                    fill: parent
+                    margins: 10
+                }
+
+                Text {
+                    width: parent.width
+                    text: "Choose Camera Control"
```

Comment:
Nitpicky detail: We tend to use sentence case across application + READMEs for titles and buttons.